### PR TITLE
[jjo] feat: update release-1.5 Jenkinsfile to match 1.6, keeping cloud releases versions

### PR DESCRIPTION
**Pull request by jjo**

### CODE REVIEW

**Comment by javsalgar** [comment]
It seems that there's a missing API in generic version 1.14

```


Error: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
```

**Comment by jjo** [comment]
> It seems that there's a missing API in generic version 1.14
> 
> ```
> 
> 
> Error: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
> ```

hm likely from kube-state-metrics image update, will back-off the images update in this PR, and leave only the Jenkinsfile bits, to do a more fine-grained images update in a followup PR

**Comment by jjo** [comment]
@javsalgar @dbarranco @jbianquetti-nami tests passing fine now PTAL

File: Jenkinsfile [hunk 1]
```diff
@@ -103,6 +140,10 @@ def waitForRollout(String namespace, int rolloutTimeout, int postRollOutSleep) {
                 """
             } catch (error) {
                 sh "kubectl --namespace ${namespace} get po,deploy,svc,ing"
+                sh """
+                  echo -n "\nFurther debugging info for Pods not-Running in '${namespace}' namespace:"
```
Comment (dbarranco) [inline Jenkinsfile:144]: this is going to help a lot, nice

**Comment by jjo** [comment]
bors bot merged commit 71bf11d into release-1.5 now \o/ :tada: 
We now have all active branches protected from exploding concurrency by locks.

**Pull request merged**